### ISSUE TITLE
Thank you very much for your excellent work, but I would like to ask where you have split your training set and test set.

### ISSUE THREAD

**Issue by WHDworld** [yyxssm/PointSSC#6]
Thank you very much for your excellent work, but I would like to ask where you have split your training set and test set.

**Comment by yyxssm** [comment]
Sorry, could you please clarify your question in more detail? It seems to be the same issue as Issue #5 .
